### ISSUE TITLE
Link Prediction via TabPFN

### ISSUE THREAD

**Issue by Demirrr** [dice-group/dice-embeddings#352]
# From Graph To Table

| Entity | LivesIn | IsA | Label
|:---|:---|:---|:---|
| CaglarDemir | Germany | Person| 1.0
| CaglarDemir | Germany | ComputerScientist| 1.0
| Germany | NotApplicable| Country | 1.0

## 💻 Graph Representation Variables

| Variable | Description | Contextual Interpretation |
|:---|:---|:---|
| $X_i$ | Represents the **first-hop information** based on outgoing edges from node $i$. | Aggregated and transformed features from direct neighbors of node $i$ (e.g., in a Graph Neural Network). |
| $X_{ij}$ | Represents the **syntactic representation** of component $j$. | A feature vector describing the **relation type** (e.g., `LivesIn` or `IsA`) of edge $j$ or a specific linguistic/structural feature of component $j$. |

# Negative Example Generation
TODO:

# Example Code
```python
from sklearn.datasets import load_breast_cancer
from sklearn.metrics import accuracy_score, roc_auc_score
from sklearn.model_selection import train_test_split

from tabpfn import TabPFNClassifier
from tabpfn.constants import ModelVersion

# Load data example
# X, y = load_breast_cancer(return_X_y=True)

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5, random_state=42)

# Initialize a classifier
clf = TabPFNClassifier()  # Uses TabPFN 2.5 weights, finetuned on real data.
# To use TabPFN v2:
# clf = TabPFNClassifier.create_default_for_version(ModelVersion.V2)
clf.fit(X_train, y_train)


# Predict probabilities
prediction_probabilities = clf.predict_proba(X_test)
print("ROC AUC:", roc_auc_score(y_test, prediction_probabilities[:, 1]))

# Predict labels
predictions = clf.predict(X_test)
print("Accuracy", accuracy_score(y_test, predictions))`
```

https://github.com/PriorLabs/TabPFN

**Comment by Demirrr** [comment]
Already being worked in a thesis
